CHANGELOG.rst

```diff
@@ -2,6 +2,289 @@
+* [:backend:`4711`] Fixed broken submission form row styling.
+* [:backend:`4695`] Fixed a performance issue during legacy Objects API registration
+  plugin validation.
+* [:backend:`4652`] Fixed misaligned validation errors in the form designer UI.
+* [:backend:`4658`] Fixed certain variants of ZIP files not passing validation on Windows.
+* [:backend:`4656`] Fixed a crash during validation when you have file upload components
+  inside repeating groups.
+
+**Project maintenance**
+
+* Fixed stories concerning bootstrap.
+* Removed automatic patching for `cosign_information` template tag.
+  (see **Installation** > **Upgrade details to Open Forms 3.0.0**) for all the necessary details.
+* Updated browserslist package.
+* Removed legacy component translations serializer fields.
+* [:backend:`4920`] Cleaned up and squashed migrations where this was possible.
+* De-duplicated Open Forms version upgrade path checks.
+* Bumped Jinja2 to 3.1.5.
+* [:backend:`4785`] Upgraded django-digid-eherkenning to 0.18.0 and updated documentation.
+* [:backend:`3283`] Removed deprecated code (please review the instructions in the documentation
+  under **Installation** > **Upgrade details to Open Forms 3.0.0** for all the necessary details):
+
+    - ``registration_backend`` and ``registration_backend_options`` fields from form.
+    - Old API location url.
+    - Conversion of ``stuf-zds-create-zaak:ext-utrecht`` to ``stuf-zds-create-zaak`` during import.
+    - Objecttype URL to UUID import conversion.
+    - Backwards compatible styling.
+    - Password Formio component.
+    - Legacy formio translation converter.
+    - Deprecated/disabled legacy OIDC callback endpoints by default.
+    - Documented registration backend migration procedure.
+    - Made Objects API and ZGW APIs group fields non-nullable where this is necessary.
+    - Normalized API endpoints to use kebab-case instead of snake-case.
+    - Removed unnecessary filter behaviour on form definitions endpoint.
+    - Removed legacy machtigen context.
+    - Removed old appointments flow and refactored code according to the new one.
+    - Made submission in temporary file uploads non-nullable.
+    - Removed conversion of form step URL to form step UUID.
+    - Made form definition name read only.
+* Documented expired domains for VCR testing.
+* [:backend:`4920`] Updated backend and frontend translations.
+* Improved test flakiness.
+* [:backend:`3457`] Extended type checking to most of the payments app.
+* Removed migration tests which relied on real models.
+* Addressed warnings in DMN components.
+* Removed duplicated MS Graph stories/plugin options.
+* Removed unused uiSchema property from registration fields.
+* Deleted obsoleted `.admin-fieldset` styling.
+* Removed the custom helptext-as-tooltip styling and applied the default styling of Django.
+* Replaced ``summary`` tag implementation with ``confirmation_summary``.
+* Bumped Django to 4.2.17 patch release.
```

Comment:
Might be good to do list item "* Updated backend dependencies" and then list the individual entries within that item, and do a similar thing for the frontend dependencies (if relevant)